### PR TITLE
Another worker process moved

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

**Comment by annando** [comment]
I'm making a break for some hours. Possibly someone wants to review it now?

File: src/Worker/DiscoverPoCo.php [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+use Friendica\Core\Config;
+use Friendica\Core\Worker;
+use Friendica\Database\DBM;
+use Friendica\Model\GlobalContact;
+use Friendica\Network\Probe;
+use Friendica\Protocol\PortableContact;
+
+require_once 'include/datetime.php';
+
+class DiscoverPoco {
```
Comment (zeroadam) [inline src/Worker/DiscoverPoCo.php]: Does it matter that all of your Worker::add calls have DiscoverPoCo but this is named DiscoverPoco?

Comment (MrPetovan) [reply src/Worker/DiscoverPoCo.php]: I second that, and suggests `DiscoverPortableContact`.

Comment (zeroadam) [reply src/Worker/DiscoverPoCo.php]: I see another conversation between you and @annando about long names in your future!! :laughing:

Comment (annando) [reply src/Worker/DiscoverPoCo.php]: What about "DiscoverPortableContactsInTheNetworkAndTheWholeInternet"? :-)

Comment (zeroadam) [reply src/Worker/DiscoverPoCo.php]: :rofl:

Comment (annando) [reply src/Worker/DiscoverPoCo.php]: Thing is: This class should be replaced anyway with separate classes.

Comment (MrPetovan) [reply src/Worker/DiscoverPoCo.php]: Nice deflection!

**Pull request merged**